### PR TITLE
finished math game, first time making a fork, hopefully works

### PR DESCRIPTION
**Pull request by heartingNinja**

### CODE REVIEW

File: MathGame.heartingNinja/MathGame.heartingNinja/GameEngine.cs [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    {
+        public int low;
+        public int high;
+        internal void AdditionGame(string message)
+        {
+            Console.WriteLine(message);
+
+            Random random = new Random();
+            int score = 0;
+
```
Comment (TheCSharpAcademy) [inline MathGame.heartingNinja/MathGame.heartingNinja/GameEngine.cs:13]: Please remove unnecessary empty line 13.

File: MathGame.heartingNinja/MathGame.heartingNinja/GameEngine.cs [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+            int firstNumber;
+            int secondNumber;
+
+            for (int i = 0; i < 5; i++)
+            {
+
```
Comment (TheCSharpAcademy) [inline MathGame.heartingNinja/MathGame.heartingNinja/GameEngine.cs:20]: Please remove unnecessary empty line 20.

File: MathGame.heartingNinja/MathGame.heartingNinja/GameEngine.cs [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+            if(high == 99)
+            {
+                Helpers.AddToHistory(score, Models.GameType.Addition, Models.GameDifficulty.Medium);
+            }
+
+            if(high == 999)
+            {
+                Helpers.AddToHistory(score, Models.GameType.Addition, Models.GameDifficulty.Hard);
+            }
+
```
Comment (TheCSharpAcademy) [inline MathGame.heartingNinja/MathGame.heartingNinja/GameEngine.cs:70]: Please remove unnecessary empty line 70

File: MathGame.heartingNinja/MathGame.heartingNinja/GameEngine.cs [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+            Random random = new Random();
+            int score = 0;
+
+
+            int firstNumber;
+            int secondNumber;
+
+            for (int i = 0; i < 5; i++)
+            {
+
```
Comment (TheCSharpAcademy) [inline MathGame.heartingNinja/MathGame.heartingNinja/GameEngine.cs:87]: Please remove unnecessary empty line 87.

File: MathGame.heartingNinja/MathGame.heartingNinja/Helpers.cs [hunk 5]
```diff
@@ -0,0 +1,87 @@
+﻿using MathProject.Models;
+using System.Linq;
+
+namespace MathProject
+{
+    internal class Helpers
+    {
+
```
Comment (TheCSharpAcademy) [inline MathGame.heartingNinja/MathGame.heartingNinja/Helpers.cs:8]: Remove unnecessary lines 8, 10, 11, 12

File: MathGame.heartingNinja/MathGame.heartingNinja/Helpers.cs [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+                result = Console.ReadLine();
+            }
+            return result;
+        }
+
+        internal static int[] GetDivisionNumbers(int low, int high)
+        {
+        
+
+        Random random = new Random();
```
Comment (TheCSharpAcademy) [inline MathGame.heartingNinja/MathGame.heartingNinja/Helpers.cs:28]: Please fix indentation in line 28

File: MathGame.heartingNinja/MathGame.heartingNinja/Helpers.cs [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+            {
+                Console.WriteLine("Your answer needs to be an integer. Try again.");
+                result = Console.ReadLine();
+            }
+            return result;
+        }
+
+        internal static int[] GetDivisionNumbers(int low, int high)
+        {
+
```
Comment (TheCSharpAcademy) [inline MathGame.heartingNinja/MathGame.heartingNinja/Helpers.cs:26]: Please remove unnecessary empty line 26 and 27

**Comment by TheCSharpAcademy** [comment]
@heartingNinja could you please submit your project on the dashboard or project page, via the "submit project" button? Thank you!

**Comment by heartingNinja** [comment]
Ok I made those changes and some others to try to follow the rules in https://www.thecsharpacademy.com/article/58 Will submit now.

**Comment by TheCSharpAcademy** [comment]
> Ok I made those changes and some others to try to follow the rules in https://www.thecsharpacademy.com/article/58 Will submit now.

I'm approving the project now, for the next ones I'll be a bit more strict with conventions. Example: The indentation is messed up in menu.cs: 

image

Also you left an unused using statement in helpers.cs:

image

You can fix those by clicking here: 

image

**Comment by TheCSharpAcademy** [comment]
@heartingNinja also could you please read and mark the "Getting Help" article as read? So I can upgrade your belt 😁

**Pull request merged**